README.md

```diff
@@ -45,6 +45,10 @@ generate this data from a wikidata dump.
 
 ## Release notes
 
+### 2.3.0 (2015-06-26)
+* Replace use of Wikibase\TermIndex::getMatchingIDs with Wikibase\TermIndex::getTopMatchingTerms.
+* EntitySelector no longer passes "type" parameter to wbsgetsuggestions which avoids an "Unrecognized parameter" warning.
```

Comment:
I meant that old versions of Wikidata that were requiring PropertySuggestor ~2.2.0 would be broken with this change, so to not break old ones we need to at least do a 0.1.0 step. If we were exact It should have been a 1.0.0 step so as to not break hypothetical users requiring ~2.2.
All that is only because PropertySuggestor does not declare the Wikibase version it is compatible with and we are not actually versioning Wikibase. If we were declaring our requirements correctly we could do either a 0.0.1 or a 0.1.0 step for this change without creating a situation where a successful composer run results in a fatal PHP error at runtime.